shell.nix

```diff
@@ -4,15 +4,27 @@ in
 { system ? builtins.currentSystem
 , config ? {}
 , pkgs ? (import (localLib.fetchNixPkgs) { inherit system config; })
-, cluster ? "staging"
+, cluster ? "demo"
 , autoStartBackend ? false
+, systemStart ? null
```

Comment:
maybe use runCommand to get current time and add 60 seconds so users don't have to specify?